docs/index.yml

```diff
@@ -263,11 +263,11 @@ additionalContent:
       # Card
       - title: Internet of things (IoT)
         links:
-          - url: /dotnet/iot/
+          - url: ./iot/index.yml
```

Comment:
The iot folder is not in this repo, so I don't think these are valid links. I guess they aren't validated by the build.

Reply:
They are in this repo, which is why it's valid.